src/components/ICA/_index.scss

```
@@ -27,13 +27,57 @@
   & &__total {
     @include carbon--type-style($name: body-short-01);
     margin-bottom: 0;
+    padding-top: $spacing-01 + $spacing-04;
```

We have hit consumer Sass conflicts in the past for not namespacing dependency spacing variables - here's an example of how this can be applied:

```suggestion
    padding-top: $carbon--spacing-01 + $carbon--spacing-04;
```